accounting/serializers.py

```diff
@@ -16,4 +21,46 @@ class Meta:
             'channel_name',
             'date_created',
             'date_updated'
-        )
\ No newline at end of file
+        )
+
+class ProjectDetailsSerializer(serializers.ModelSerializer):
+    """ project serializer
+    """
+    date_data = serializers.SerializerMethodField()
+    pending_issues = serializers.SerializerMethodField()
+
+    class Meta:
+        model = Project
+        fields = (
+            'id',
+            'name',
+            'date_data',
+            'description',
+            'channel_name',
+            'date_created',
+            'date_updated',
+            'pending_issues'
+        )
+
+    def get_date_data(self, obj):
+        from history.serializers import ShortStandupProjectSerializer
+
+        today = datetime.now().date()
+        start_of_week = today - timedelta(days=today.weekday())
+        end_of_week = start_of_week + timedelta(days=7)
+        
+        result = {
+            'start_of_week': start_of_week,
+            'end_of_week': end_of_week - timedelta(days=1)
+        }
+
+        return result
+    
+    def get_pending_issues(self, obj):
+        #find better solution
```

Comment:
#solution.
1. delete this as this is redundant.
2. trace the code a bit more (dig deeper)
`feed.views.Notification.pending()` <-- similar just replicate

Reply:
i'll give you a hint.

try this.
```
blockers = Blocker.objects.filter(standup__in=obj.standup_set.all())
```

Reply:
by the way, don't name the project instance as `obj`. it took me a few minutes what is that `obj`